test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -1417,6 +1417,7 @@ public static IEnumerable GetStreamCases()
 
             using var conn = await OpenConnectionAsync();
             using var cmd = new NpgsqlCommand("SELECT @p, @p", conn) { Parameters = { new NpgsqlParameter("p", value.Generic) } };
+            cmd.CommandTimeout = 5;
```

Comment:
Why is this needed?

Reply:
Because if there is still a bug, it will be stuck forever.

Reply:
OK. I think it's OK for a test to be stuck forever if there's a bug in Npgsql - this can happen with most of our tests in any case, probably no need to guard against this.

Reply:
Up to you, I don't mind leaving it in.